python/semantic_kernel/memory/volatile_memory_store.py

```diff
@@ -32,10 +34,7 @@ async def get_nearest_matches_async(
 
         # Use numpy to get the cosine similarity between the query
         # embedding and all the embeddings in the collection
-        similarity_scores = (
-            embedding.dot(embedding_array.T)
-            / (linalg.norm(embedding) * linalg.norm(embedding_array, axis=1))
-        )[0]
+        similarity_scores = self.compute_similarity_scores(embedding, embedding_array)
```

Comment:
If `compute_similarity_scores` is `async` we should `await` it here, but I'd rather you just make it non-async

Reply:
Done